PssNationalInstance/src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 server.port=7001
-spring.profiles.active=local
+spring.profiles.active=localdev
```

Comment:
This ping-pong approach is not sustainable and requires a rethink @nelsonkimaiga. 

I suggest that this is handled at runtime on the docker-compose(https://github.com/IntelliSOFT-Consulting/PSS-Insight-v2-Deployment-international/blob/main/international/docker-compose.yaml) by either setting the relevant environment variable (SPRING_PROFILES_ACTIVE) or passing the active profile name at runtime (see https://stackoverflow.com/a/38520863). 

This will allow for the flexibility of being able to set the active profile at the environment level.

Reply:
Have put these additions in the deployment package: https://github.com/IntelliSOFT-Consulting/PSS-Insight-v2-Deployment-national/pull/21